fabric-biomes-v1/build.gradle

```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-biomes-v1"
-version = getSubprojectVersion(project, "0.1.0")
+version = getSubprojectVersion(project, "0.1")
```

Comment:
Version that end in either `.0` or `.1` were left alone as they would all become `.1` and not be "going back"